src/components/ui/register/steps/StepEmail.tsx

```
@@ -0,0 +1,37 @@
+import * as S from '@/containers/register/style';
+import { RegisterFormData } from '@/types/register';
+import Square from '@/components/ui/button/square';
+import { memo } from 'react';
+
+interface Props {
+  form: RegisterFormData;
+  onChange: (k: keyof RegisterFormData, v: any) => void;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)